### DISCUSSION TITLE
depotdownloader opens and then closes immediately

### DISCUSSION THREAD

**Discussion by SenrirenTokyo** [SteamRE/DepotDownloader#495] [Q&A]
### What did you expect to happen?

The app to actually work and not just crash

### Instead of that, what actually happened?

I opened the app, and it immediately closed, downloaded 2.50, downloaded windows, neither worked. added the 2.50 dll's and stuff to a folder with the windows x64 .exe, still didnt work

### Which operating system are you running on?

Windows 11

### Version

2.50

**Comment by voided** [comment]
How are you running it? DepotDownloader is a terminal/console application - there is no GUI.

See also: https://github.com/SteamRE/DepotDownloader/discussions/440

**Comment by ABakedWaffle** [comment]
I'm having the exact same issue. I downloaded the separate depotdownloaders for my system, then upon opening the downloader, the command terminal opens and immediately closes without downloading anything

**Comment by Keltas** [comment]
You need to run the program through a command line. To do it the fastest way would be to open the folder where it is downloaded, shift+right-click on the empty space and select "open powershell window here". Then you are typing the name of the program (DepotDownloader.exe) and it should tell you what command you need to type to get what you want. 
For example:  ".\DepotDownloader.exe -app (insert appid) -depot (insert depot ID) -manifest (insert manifest ID) -username (insert your steam username)"
You will then need to type in your Steam password and then code from your steam 2 factor authentication. Afterwards the files should start downloading.

**Comment by zdl200632929** [comment]
PS D:\wz> DepotDownloader.exe
DepotDownloader.exe : 无法将“DepotDownloader.exe”项识别为 cmdlet、函数、脚本文件或可运行程序的名称。请检查名称的拼写
，如果包括路径，请确保路径正确，然后再试一次。
所在位置 行:1 字符: 1
+ DepotDownloader.exe
+ ~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (DepotDownloader.exe:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException


Suggestion [3,General]: 找不到命令 DepotDownloader.exe，但它确实存在于当前位置。默认情况下，Windows PowerShell 不会从当 前位置加载命令。如果信任此命令，请改为键入“.\DepotDownloader.exe”。有关详细信息，请参阅 "get-help about_Command_Precedence"。
PS D:\wz>
